pkg/fission-cli/logdb/kubernetes_log.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2016 The Fission Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logdb
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+
+	fv1 "github.com/fission/fission/pkg/apis/core/v1"
+	"github.com/fission/fission/pkg/fission-cli/cmd"
+)
+
+type LogDBOptions struct {
+	Client cmd.Client
+}
+
+type kubernetesLogs struct {
+	ctx    context.Context
+	client cmd.Client
+}
+
+func (k kubernetesLogs) GetLogs(logFilter LogFilter) (podLogs *bytes.Buffer, err error) {
+	podLogs, err = GetFunctionPodLogs(k.ctx, k.client, logFilter)
+	return podLogs, err
+}
```

Comment:
1. We get proper logs for pool manager based pods
2. `-f` flag help us to watch logs. And we fetch for newer logs every second. Which makes sure only new logs are printed
3. Continuously run requests against function- Works fine. New logs flow in with -f flag.
4. **Not sure about this.**  Need more testing here.
5. Since we fetch pod list before each log stream call, we stay updated about newer pods. And are able to fetch logs for the same.